infra/test/integration/simple_example/simple_example_test.go

```diff
@@ -73,8 +80,16 @@ func TestSimpleExample(t *testing.T) {
 				annotateGcsFunctionRegion = eventTrigger.Get("triggerRegion").String()
 				fmt.Printf("triggerRegion: %s\n", annotateGcsFunctionRegion)
 			}
+			if strings.Contains(functionName, "functions/" + annotateHttpFunctionName) {
```

Comment:
nit: do string concatenation before conditional

Reply:
Thanks for your suggestion. It has been fixed in [c3f370d](https://github.com/GoogleCloudPlatform/terraform-ml-image-annotation-gcf/pull/28/commits/c3f370dc3e5ad7baff1afa7e0600ccc24cc4bfc7)